tests/update/patch_before_migration.pm

```diff
@@ -0,0 +1,97 @@
+# Patch SLE11* qcow2 images before migration (offline)
```

Comment:
that's somehow wrong, you also patch sle12, right? better just delete this line and optionally add something to '# Summary: ' further below